### PR TITLE
Update Daily Check In and add DB Version Check

### PR DESCRIPTION
**Pull request by MarkDavies0**
## Changes
Add simplified `dailyCheckIns` collection in firebase as well as corresponding screens. Also adds a version check with firebase under the `version` collection in Firestore. Add `SUPPORTED_DB_VERSION=1.0` to the end of your .env file

## PR Dependencies
Depends on: #136 

## Requirements
run the setup script which runs the updated `env_set.bash`

## Testing

- [x] Manual

### CODE REVIEW

**Comment by wynnset** [comment]
We should change the question to be "It's time for your daily check-in" and make sure it only appears once per day:
image

File: common/models/Moods.ts [hunk 1]
```diff
@@ -29,8 +29,25 @@ export namespace Moods {
...
+                case 'very bad':
+                    return 'not at all';
+                case 'bad':
+                    return 'not really';
+                case 'ok':
+                    return 'mostly';
+                case 'good':
+                    return 'pretty good';
+                case 'very good':
+                    return 'yes';
```
Comment (wynnset) [inline common/models/Moods.ts:45]: Laura is working to finalize these labels, but we can keep them like this for now.

File: mobile/src/stateMachine/views/main/home.tsx [hunk 2]
```diff
@@ -399,33 +406,39 @@ export class HomeView extends ViewState<{ opacity: Animated.Value, isUnfinishedQ
...
+                                    :
+                                    <TouchableNativeFeedback onPress={this.onTapOrb} style={{ width: Layout.window.width, height: this.orbTapContainerHeight, zIndex: 9999 }}>
+                                        <View style={{ width: '100%', height: this.orbTapContainerHeight }} />
+                                    </TouchableNativeFeedback>
+                                }
+                            </Portal>
+                            {/* MK-TODO below buttons used for development/testing only and will be removed */}
+                            {/* <View style={{ flexDirection: 'row' }}>
+                    <Button title='TESTING' style={styles.testingButton} onPress={() => this.onTESTINGButton()} theme={this.theme} />
+                </View> */}
```
Comment (wynnset) [inline mobile/src/stateMachine/views/main/home.tsx:432]: Can we remove this now? Or are we still using it?

File: mobile/src/viewModels/VersionViewModel.ts [hunk 3]
```diff
@@ -0,0 +1,28 @@
+import { VersionStub } from '../../../common/controllers/VersionController';
+import AppController from 'src/controllers';
+
+const SUPPORTED_DB_VERSION = '1.0.3';
```
Comment (wynnset) [inline mobile/src/viewModels/VersionViewModel.ts:4]: Can we move this to the `.env` file and use that env variable here instead? (or is this done through the template file already? if so, can we put this version as 1.0 instead)

**Review by wynnset — COMMENTED** [review]
Rather than commenting out the content of those view files, can we just comment out where we import them instead? Not a big deal if it's really difficult.

**Comment by MarkDavies0** [comment]
> Rather than commenting out the content of those view files, can we just comment out where we import them instead? Not a big deal if it's really difficult.

I commented them all the way out so that if you do a global find it's clear what code is/isn't being used

File: mobile/src/viewModels/VersionViewModel.ts (hunk 3 above)
Comment (MarkDavies0) [reply mobile/src/viewModels/VersionViewModel.ts:4]: Yeah this file is generated from the template, ill set to 1.0

File: mobile/src/stateMachine/views/main/home.tsx (hunk 2 above)
Comment (MarkDavies0) [reply mobile/src/stateMachine/views/main/home.tsx:432]: nope ill remove

**Review by wynnset — APPROVED** [review]

**Pull request merged**